### ISSUE TITLE
Need a method for doing an `oc tag` based promotion

### ISSUE THREAD

**Issue by etsauer** [redhat-cop/pipeline-library#13]
@redhat-cop/developer-workflow

**Comment by syvanen** [comment]
There's actually not much magic to this:

```
    stage("Promote image") {
      steps {
        script {
          openshift.withCluster() {
            openshift.tag("${STAGE1}/${APP_NAME}:latest", "${STAGE2}/${APP_NAME}:latest")
          }
        }
      }
    }
```

**Comment by pabrahamsson** [comment]
Fixed by #16
